src/ProjectTemplates/test/MvcTemplateTest.cs

```diff
@@ -11,16 +12,22 @@ public class MvcTemplateTest
     {
         public MvcTemplateTest(ProjectFactoryFixture projectFactory, ITestOutputHelper output)
         {
-            Project = projectFactory.CreateProject(output);
+            ProjectFactory = projectFactory;
+            Output = output;
         }
 
-        public Project Project { get; }
+        public Project Project { get; set; }
```

Comment:
Again, no point in this if its only use is in the test that created it.

Reply:
This will be used when I break the test into multiple stages.